blueocean-core-js/package.json

```diff
@@ -5,8 +5,7 @@
   "main": "dist/js/index.js",
   "scripts": {
     "gulp": "gulp",
-    "test": "gulp test",
-    "prepublish": "gulp test; gulp"
```

Comment:
anytime you do `npm install` _locally_, no? If you're actually doing `npm install` from npm is it doesn't actually run this, right? Seems like we probably would still want it...